### PR TITLE
Note that we do not define trust properties because we do not define …

### PR DESCRIPTION
**Pull request by cyrill-k**
Note that we do not define trust properties because we do not define a threat model

### CODE REVIEW

File: draft-irtf-panrg-path-properties.md [hunk 1]
```diff
@@ -241,6 +241,7 @@ If nodes are basing policy or path selection decisions on path properties, they
 In order to be able to trust such path properties, nodes may need to establish a trust relationship or be able to verify the authenticity, integrity, and correctness of path properties received from another node.
 
 Security related properties such as confidentiality and integrity protection of payloads are difficult to characterize since they are only meaningful with respect to a threat model which depends on the use case, application, environment, and other factors.
+Likewise, properties for trust relations between nodes cannot be meaningfully defined without a concrete threat model.
```
Comment (theri) [inline draft-irtf-panrg-path-properties.md:244]:
```suggestion
Likewise, properties for trust relations between nodes cannot be meaningfully defined without a concrete threat model, and defining a threat model is out of scope for this draft.
```

File: draft-irtf-panrg-path-properties.md [hunk 2]
```diff
@@ -241,6 +241,7 @@ If nodes are basing policy or path selection decisions on path properties, they
...
 Such properties are orthogonal to the path terminology and path properties defined in this document, as they are tied to the communicating entities and protocols used (e.g., client and server using HTTPS, or client and remote network node using VPN) and the path is typically oblivious to these properties.
```
Comment (theri) [inline draft-irtf-panrg-path-properties.md:245]:
```suggestion
Properties related to confidentiality, integrity, and trust are orthogonal to the path terminology and path properties defined in this document, as they are tied to the communicating entities and protocols used (e.g., client and server using HTTPS, or client and remote network node using VPN) and the path is typically oblivious to these properties.
```
Trying to make it go well with the sentence before.
Also, this sentence is rather long, maybe we should split it? But I can't immediately see a good way to do so. Doesn't have to be in this PR.

Comment (cyrill-k) [reply draft-irtf-panrg-path-properties.md:245]: I like the changes.
i will split the long sentence in another commit

**Review by cyrill-k — COMMENTED** [review]
Security Considerations Textual changes

**Comment by cyrill-k** [comment]
I rephrased the long sentence.
What do you think?

**Review by theri — APPROVED** [review]
Looks good to me! Thank you.

**Pull request merged**